OpenRA.Game/Graphics/IGraphicsDevice.cs

```diff
@@ -135,6 +137,6 @@ public enum WindowMode
 	{
 		Windowed,
 		Fullscreen,
-		PseudoFullscreen,
+		NativeFullscreen,
```

Comment:
You've introduced a terminology change here.  This should be at least in its own commit, but preferably in its own pr.